src/utils/useChromaticDialog.ts

```diff
@@ -0,0 +1,58 @@
+import { useCallback, useEffect, useRef } from "react";
+import { z } from "zod";
```

Comment:
I feel like it's a bit heavy-handed to introduce zod just for this purpose. Do you have other uses in mind?

Reply:
It's already a dependency of the CLI, so I don't think that's a big issue?